<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTBrowser</class>
 <widget class="QMainWindow" name="QTBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTBrowser</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="opaqueResize">
       <bool>true</bool>
      </property>
      <property name="handleWidth">
       <number>2</number>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="minimumSize">
           <size>
            <width>400</width>
            <height>0</height>
           </size>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="tab_2">
           <attribute name="title">
            <string>Download Files</string>
           </attribute>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="DQMFileDownloader" name="widget_2" native="true"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab">
           <attribute name="title">
            <string>View Downloaded Files</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="TFileTreeViewer" name="widget_3" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="SuperimposePlugin" name="widget" native="true"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRemote_Files">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuPlugins">
    <property name="title">
     <string>Pl&amp;ugins</string>
    </property>
    <addaction name="actionAbout_Plugins"/>
   </widget>
   <addaction name="menuRemote_Files"/>
   <addaction name="menuPlugins"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSettings">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
  </action>
  <action name="actionAbout_Plugins">
   <property name="text">
    <string>&amp;About Plugins</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SuperimposePlugin</class>
   <extends>QWidget</extends>
   <header>plugins/superimposeplugin.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DQMFileDownloader</class>
   <extends>QWidget</extends>
   <header>dqmfiledownloader/dqmfiledownloader.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TFileTreeViewer</class>
   <extends>QWidget</extends>
   <header>tfiletreeviewer/tfiletreeviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
